draft-parecki-oauth-client-id-metadata-document.md

```diff
@@ -166,15 +165,23 @@ The client metadata document MAY define additional properties in the response.
 The client metadata document MAY also be served with more specific content types
 as long as the response is JSON and conforms to `application/<AS-defined>+json`.
 
-The `token_endpoint_auth_method` property MUST NOT include `client_secret_post`
-or `client_secret_basic`, as there is no way to establish a shared secret to be
-used with these authentication methods. See {{client_authentication}} for more details.
+As there is no way to establish a shared secret to be used with client metadata
+documents, the following restrictions apply on the contents of the
+client metadata document:
+
+* the `token_endpoint_auth_method` property MUST NOT include `client_secret_post`
+or `client_secret_basic`
+* the `client_secret_expires_at` property MUST NOT be used
```

Comment:
As far as I know, this isn't a relevant property, and `private_key_jwt` as mentioned in the Client Authentication Security Consideration does not have a "expires at", and `client_secret_expires_at` only is relevant for the `client_secret` property.